internal/policies/mount/mount_test.go

```diff
@@ -103,23 +147,40 @@ func TestApplyPolicy(t *testing.T) {
 		"error when cleaning up user policy with no entries and path already exists as a directory":  {entries: []string{"no entries"}, pathAlreadyExists: true, wantErr: true},
 		"error when cleaning up user policy with empty entry and path already exists as a directory": {entries: []string{"entry with no value"}, pathAlreadyExists: true, wantErr: true},
 		"error when applying policy with entry containing badly formatted value":                     {entries: []string{"entry with badly formatted value"}, wantErr: true},
-	}
 
-	u, err := user.Current()
-	require.NoError(t, err, "Setup: failed to get current user")
+		/**************************** SYSTEM ***************************/
+		// Error cases.
+		"error when creating units with bad entry values":                        {entries: []string{"entry with badly formatted value"}, isComputer: true, wantErr: true},
```

Comment:
So, we have all those error cases which is great, but I think some non errors are missing which is our contract:
- if stop is failing, we still want disable to do its work
- we don’t error out if start is failing
The current tests don’t cover those if I’m correct?

Reply:
Another part of missing test to me: daemon-reload failing.

You should be able to confirm this with coverage report anyway, did you run it and assess the parts that were not covered?

Reply:
> So, we have all those error cases which is great, but I think some non errors are missing which is our contract:
> 
> * if stop is failing, we still want disable to do its work
> * we don’t error out if start is failing
>   The current tests don’t cover those if I’m correct?

We do check when start fails, the test case name is `only emit a warning when starting new units and systemctl fails`.

Reply:
there is the stop one as we will separate it to do, but the rest is done. As I opened a new comment, closing this one.